src/itemEditableInfo/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import { TextDisplayType } from 'text'
+import Item from '_utils/item'
+import { A11yProps, pickA11yProps } from '_utils/interfaces'
+import { color } from '_utils/branding'
+
+export interface ItemEditableInfoProps extends A11yProps {
+  readonly className?: Classcat.Class
```

Comment:
After speaking with @simonrelet we have no reason to have this type, `className` should always be a string
`readonly className?: string`